src/hooks/useChapterId.ts

```diff
@@ -53,9 +54,11 @@ const useChapterIdsByUrlPath = (lang: string): string[] => {
         setChapterIds(chapterIdsForPage);
       } else if (juzId) {
         setChapterIds(await getChapterIdsForJuz(formatStringNumber(juzId as string)));
-      }
+      } // else if (hizbId) {
```

Comment:
Since we are not using this anymore, I suggest reverting this page hook back to how it was.

Reply:
Sure, but I thought we'd need it in the future because we're using this hook in `components/AudioPlayer/Buttons/PlayPauseButton.tsx`. I haven't tested the audio player, yet.

Reply:
Oh sorry let me think of it. Will get back to you inshAllah